src/textual/app.py

```diff
@@ -389,6 +414,16 @@ async def process_messages(self) -> None:
         log("---")
         log(f"driver={self.driver_class}")
 
+        if os.getenv("TEXTUAL_DEVTOOLS") == "1":
+            try:
+                await self.devtools.connect()
+                self.log_file.write(f"Connected to devtools ({self.devtools.url})\n")
+                self.log_file.flush()
```

Comment:
Do you need these `flush()` calls? The log file should be line buffered, so will be effectively flushed already.

Reply:
Nope, removed them